auction.py

```diff
@@ -56,6 +56,10 @@ def __init__(self, client, debug=False):
       181700384279101440,  # fspoon
     ]
     self.league = 'PST'                   #enter league name
+    self.emojis = [
+      '<:plus:913878220489773099>',       #plus for bot reaction
+      '<:minus:913877912355229727>'       #minus for bot reaction
+    ]
```

Comment:
this probably shouldn't be a list, since elsewhere in the code you do like "react with `self.emojis[0]`" and it's hard to figure out from context what that means. This should probably be a dictionary like `{'plus': <:.....>}` and then in the code you can do `react with self.emojis['plus']` or something like that

Reply:
Updated to a dict with 'plus'/'minus' keys